src/components/Table/GridTable.tsx

```diff
@@ -49,7 +49,16 @@ export interface GridStyle {
   firstNonHeaderRowCss?: Properties;
   /** Applied to all cell divs (via a selector off the base div). */
   cellCss?: Properties;
-  /** Applied to the header (really first) row div. */
+  /**
+   * Applied to the header (really first) row div.
+   *
+   * NOTE: Adding margin bottom to a virtual table header will not result in the
+   * expected of adding space between the header and list. This will only add
+   * margin bottom to the cell itself and now the header row.
+   *
+   * To achieve space between the header and list, add margin top to the
+   * `firstNonHeaderRowCss` styles.
+   * */
   headerCellCss?: Properties;
```

Comment:
Added this comment to help with common Virtual GridTable style issues where the expectation is that this will add space between the header and list. This will not since a virtual table has two separate adjacent `divs` for the header list and the list(non-header) list.